sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -81,6 +81,25 @@ public static BinaryData Write(object model, ModelReaderWriterOptions? options =
         }
     }
 
+    /// <summary>
```

Comment:
Overall I think this is a good addition but will let @KrzysztofCwalina and @annelo-msft sign off on the public API surface.